code/datums/components/rot.dm

```diff
@@ -30,6 +39,7 @@
 
 /datum/component/rot/corpse
 	amount = MIASMA_CORPSE_MOLES
+	time_remaining = 7 MINUTES //2 minutes more to compensate for the delay
```

Comment:
Doesn't the time only tick down once the corpse process reaches line 71 though? So the 2 minute "time since death" cap, wouldn't affect this?

Reply:
Oh you're right, dang it got merged already